frontend/src/components/dashboard/ArtworkCloseUpCard.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import AudioPlayer from 'react-audio-player';
+import PlaceholderImage from './images/paint.jpg'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    flexDirection: "row"
```

Comment:
I think we are supposed to use single quotes in the styles part but could be mixing it up

Reply:
yes, you're right. I ended up fixing this in the next PR when I added ESLint